### ISSUE TITLE
NTR:  septated mononucleate vegetative cell with misplaced nucleus

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3906]
parent of
inviable septated mononucleate vegetative cell

child of
 mononucleate vegetative cell with mislocalized nucleus

standard def
PMID:33506191

I previously assumed that all "septated mononucleate vegetative cell" would b e inviable, but this is not the case.

If this is observed in ts mutants, the mononucleate cells can go on to divide as diploids!

**Comment by ValWood** [comment]
Also, this term

Definition | Parents
An inviable phenotype observed in the vegetative growth phase of the life cycle in which a cell undergoes septation despite abolished mitotic sister chromatid separation, producing inviable daughter cells, and in which the septum forms in a position that partitions the nucleus into one compartment. Cell separation may or may not be completed.

Is the same thing. At one point  we decided to call this  "cut" because it accompanies 'cut' but this is not a 'cut'  because the nucleus is not bisected
.
Illustrated really nicely here:
The nucleus is either 'displaced" or it is 'cut'

<img width="861" alt="Screenshot 2021-02-10 at 18 56 29" src="https://user-images.githubusercontent.com/7359272/107557624-c4dbad00-6bd1-11eb-8e64-deb25920821f.png">
 

so we should  probably turn this into the new term I suggested above, but removing 
references to cut

**Comment by ValWood** [comment]
Finally, we probably need to do some merging here:

These are all descendants of 'cut' in Canto
<img width="397" alt="Screenshot 2021-02-09 at 21 59 23" src="https://user-images.githubusercontent.com/7359272/107558007-374c8d00-6bd2-11eb-86ea-936939327886.png">

mitotic catastrophe, elongated cell -> mitotic catastrophe with cut, elongated cell 

IF mitotic catastrophe is always 'cut' 
(cut is always a mitotic catastrophe)

First I need to find out if a mitotic catastrophe is always a cut, or if there are other ways to bgeneragte a mitotic catastrophe.

**Comment by mah11** [comment]
Do you still need the new term originally requested here? If so, I'll hive the cut & mitotic catastrophe stuff off into its own ticket, because it's sort of a tangent (and adding the new term is a lot more straightforward).

**Comment by ValWood** [comment]
Sorry about the missing term, ow added. I always forget that if you copy past from  Pombase term page, the text above the line (the term in this instance) does not appear.

**Comment by ValWood** [comment]
I already moved MC into a new ticket. this term is still required.

**Comment by mah11** [comment]
Argh, moving the FYPO:0005070 discussion back here because it is relevant to both and the latest round is more to do with the new request ...

>> What (if any) is the action to take for FYPO:0005070?
> 
> I don't know. It depends whether you agree that it is logically equivalent to the term I requested. The term I requested is in a different place, because it is not 'cut', but the existing term FYPO:0005070 could possibly be repurposed?

Well, they aren't logically equivalent because FYPO:0005070 has paths in the process phenotype branch (via cut, and also reflecting the "abolished mitotic sister chromatid separation" part of its name and def), whereas the vast majority of the "yada-septate yada-nucleate veg cell" terms don't (they're only in the viability and physical quality branches). The new term wouldn't fall into the process branch, at least as currently proposed.

FYPO:0005070 didn't originally have "cut" in its name, and I don't remember why it got changed (I must have done the edit, but it's just gone from my brain). Most likely done at the time of #3173 ... 

Anyway, I'm now inclined to make FYPO:0005070 obsolete because of the erroneous addition of "cut".

> and make
> abolished mitotic sister chromatid separation, with mononucleate and anucleate compartment formation
> an exact synonym of the proposed new term
> "septated mononucleate vegetative cell with misplaced nucleus"

These phrases are not exactly synonymous for the same sort of reasons as above -- the "sister chromatid separation" portion adds a process angle that is absent from "septated mononucleate vegetative cell with misplaced nucleus". Stricly speaking, we could have the process term linked via has_output to the "septated ..." term, but that might be overkill.

**Comment by ValWood** [comment]
OK I agree obsoletion is best and there are not many annotations:
https://www.pombase.org/term/FYPO:0005070
I will  deal with these once the new term is in place to check if they are equivalent phenotypes.

I could  make a concurrent term to 'abolished chromosome segregation', if required.

If the cell is septated and mononucleate this is a given, but it might be a bit weird to include 'abolished chromosome segregation' in this term  because the 'abolished chromosome segregation' was  really in the previous cell cycle?

**Comment by mah11** [comment]
re comment from https://github.com/pombase/curation/issues/2955:

> abolished mitotic sister chromatid segregation, with mononucleate and anucleate compartment formation
> 
> @mah11 could the existing term be repurposed? probably not.

Already discussed - see [above](https://github.com/pombase/fypo/issues/3906#issuecomment-777442314).

> note also segregation, not separation.

So the other parts of segregation (chromosome condensation, metaphase plate congression, etc.) also don't happen? That's what this note implies.

**Comment by ValWood** [comment]
> So the other parts of segregation (chromosome condensation, metaphase plate congression, etc.) also don't happen? That's what this note implies.

Does it? But we often have things that condense normally and fail to segregate into different compartments?

Here segregation fails because both  nuclei end up in the same compartment.
This seems to be a broader failure than just 'separation' defect? Maybe separation is OK.  But separation is only the release of cohesin, and I don't know. if this happens or not...

**Comment by mah11** [comment]
> Does it? But we often have things that condense normally and fail to segregate into different compartments?

Following the arrangement of the relevant GO BP terms, these should not be annotated to "abolished chromosome segregation". Part of the segregation process not only happens, it happens normally, so the overall process is not abolished; it starts but goes wrong partway through.

> Here segregation fails because both nuclei end up in the same compartment. This seems to be a broader failure than just 'separation' defect? Maybe separation is OK. But separation is only the release of cohesin, and I don't know. if this happens or not.

Going into different compartments is chromosome separation plus subsequent parts of segregation, but not the parts of segregation that precede separation. I don't see a grouping in GO for that (and I guess we wouldn't worry about grouping segregation and subsequent events for normal happy cells).

"Fails" does not necessarily mean "abolished"; I suspect it doesn't correspond neatly to a PATO-defined quality at all. As noted above, a process can start (and is therefore not abolished) and not finish correctly.

**Comment by mah11** [comment]
OK, the new term is
  septated mononucleate vegetative cell with mislocalized nucleus FYPO:0007646

and FYPO:0005070 is obsolete. I'll close this because the original request and follow-up obsoletion are done.

**Comment by ValWood** [comment]
> "Fails" does not necessarily mean "abolished"; 

good point. I was then worried about my existing annotation, but 'abolished' is hardly used at all (and never for mitotic). Phew...
